wouso/resources/static/css/cpanel-style.css

```diff
@@ -120,3 +120,20 @@ div.second {
 .form-control {
     width: 40%;
 }
+/* Groups */
+.right {
+    text-align: right;
+}
+.btn-toggle{
+    margin-bottom: 10px;
+}
+.ctr-number{
+    font-weight: bold;
+}
+.label-3{
+    margin: 0px;
+}
+.group-row{
```

Comment:
@catalin-oancea you should remove display:none from group-row class, because it's not really related. You could add the hidden class instead.